### PR TITLE
Add agat sp add introns

### PR DESCRIPTION
**Pull request by Leila011**
## Description

add agat_sp_add_introns

Closes #xxxx 

## Checklist before requesting a review

- [x] I have performed a self-review of my code

- [x] Conforms to the [Contributing guidelines](https://github.com/viash-hub/base/blob/main/CONTRIBUTING.md)

- [x] Proposed changes are described in the [CHANGELOG.md](https://github.com/viash-hub/base/blob/main/CHANGELOG.md)

- [x] I have tested my code with `viash ns test --parallel -q <name or namespace>`

- Check the correct box. Does this PR contain:
  - [ ] Breaking changes
  - [x] New functionality
  - [ ] Major changes
  - [ ] Minor changes
  - [ ] Documentation
  - [ ] Bug fixes

### CODE REVIEW

File: src/agat/agat_sp_add_introns/config.vsh.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+name: agat_sp_add_introns
+namespace: agat
+description: |
+  The script aims to add intron features to gtf/gff file without intron features.
```
Comment (DriesSchaumont) [inline src/agat/agat_sp_add_introns/config.vsh.yaml:4]:
```suggestion
  Add intronic elements to a gtf/gff file without intron features.
```

File: src/agat/agat_sp_add_introns/config.vsh.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        type: file
+        direction: output
+        required: true
+        example: output.gff
+  - name: Arguments
+    arguments:
+      - name: --config
+        alternatives: [-c]
+        description: |
+          String - Input agat config file. By default AGAT takes as input agat_config.yaml file from the working directory if any, otherwise it takes the original agat_config.yaml shipped with AGAT. To get the agat_config.yaml locally type: "agat config --expose". The --config option gives you the possibility to use your own AGAT config file (located elsewhere or named differently).
```
Comment (DriesSchaumont) [inline src/agat/agat_sp_add_introns/config.vsh.yaml:42]:
```suggestion
          Agat config file. By default AGAT takes the original agat_config.yaml shipped with AGAT. The --config option gives you the possibility to use your own AGAT config file (located elsewhere or named differently).
```

**Review by DriesSchaumont — CHANGES_REQUESTED** [review]
Small suggestions

File: src/agat/agat_sp_add_introns/config.vsh.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+keywords: [gene annotations, GTF conversion]
+links:
+  homepage: https://github.com/NBISweden/AGAT
+  documentation: https://agat.readthedocs.io/en/latest/tools/agat_sp_add_introns.html
+  issue_tracker: https://github.com/NBISweden/AGAT/issues
+  repository: https://github.com/NBISweden/AGAT
+references: 
+  doi: 10.5281/zenodo.3552717
+license: GPL-3.0
+authors:
```
Comment (DriesSchaumont) [inline src/agat/agat_sp_add_introns/config.vsh.yaml:14]:
```suggestion
requirements:
  commands: [agat]
authors:
```

**Review by DriesSchaumont — CHANGES_REQUESTED** [review]

**Review by emmarousseau — APPROVED** [review]
Reduced test data size
Minor changes to config formatting

**Pull request merged**